src/components/SwipeThumb/index.js

```diff
@@ -144,12 +144,19 @@ class SwipeThumb extends React.Component {
       if (this.props.onSwipeSuccess) {
         this.props.onSwipeSuccess();
       }
-      // this.reset(); // Enable this line to reset the thumb after successful swipe
+
+      //Animate back to initial position
+      if (this.props.shouldResetAfterSuccess)
+        Animated.timing(this.state.animatedWidth, {
+          toValue: this.defaultContainerWidth,
+          duration: this.props.resetAfterSuccessAnimDuration || 200,
```

Comment:
If we pass 0 to prop resetAfterSuccessAnimDuration, still it will be 200. I'm gonna move this to default props.